packages/amazonq/package.json

```diff
@@ -127,6 +127,11 @@
                     "markdownDescription": "%AWS.configuration.description.amazonq%",
                     "default": true
                 },
+                "amazonQ.allowDevCommands": {
+                    "markdownDescription": "%AWS.configuration.description.allowDevCommands%",
+                    "type": "object",
```

Comment:
Is there a better name for that setting? `allowDevCommands` makes me think that its a boolean rather than an object

Reply:
sure, how does `allowDevCommandsConfiguration` sound? 

it was quite tricky for me to do the naming in the first place

Reply:
we have decided to go with `devCommandWorkspaceConfigurations`